contracts/Compass.vy

```diff
@@ -43,6 +55,11 @@ struct TokenSendArgs:
     receiver: DynArray[address, MAX_BATCH]
     amount: DynArray[uint256, MAX_BATCH]
 
+struct FeeArgs:
+    community_fee: uint256 # Total amount to alot for community wallet
+    security_fee: uint256 # Total amount to alot for security wallet
+    fee_payer_paloma_address: bytes32 # Paloma address covering the fees
```

Comment:
Is bytes32 okay for this? I think it needs to be a `string[64]`, which is used in the `SendToPalomaEvent` which contains a paloma receiver address in string notation.

Reply:
Would need to be changed at numerous points in the contract.

Reply:
I changed it deliberately. String processing requires much more gas than bytes32 and I don't think different type of the same conception looks good. If it doesn't take too much time, I'd like to use bytes32 type address only for paloma address.